src/app/components/profile/profile.component.ts

```diff
@@ -125,6 +125,12 @@ export class ProfileComponent implements OnInit {
    * Displays a Modal to update user details
    */
   updateUserDetails() {
+    const first_name = (this.user['first_name'] === '-') ? '' : this.user['first_name'];
+    const last_name = (this.user['last_name'] === '-') ? '' : this.user['last_name'];
+    const affiliation = (this.user['affiliation'] === '-') ? '' : this.user['affiliation'];
+    const google_scholar_url = (this.user['google_scholar_url'] === '-') ? '' : this.user['google_scholar_url'];
+    const github_url = (this.user['github_url'] === '-') ? '' : this.user['github_url'];
+    const linkedin_url = (this.user['linkedin_url'] === '-') ? '' : this.user['linkedin_url'];
```

Comment:
Sorry for this :smile:  but can you please use camel-case here for variable names like `firstName`?

Reply:
first_name -> FirstName
last_name -> LastName
affiliation -> Affiliation
it this okay?

Reply:
no, they should be in camel-case, like `first_name` -> `firstName` and `last_name` -> lastName